### PR TITLE
Adds Getting Started with introduction and Yeoman

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

**Comment by fernandosouza** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/pages/docs/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+<article id="2">
+
+## Architecture
+
+![metal_architecture](http://metaljs.com/images/metal_architecture@2x.png)
+
+Metal.js' main classes are State and Component. Component actually extends from State, thus containing all its features. The main difference between the two is that Component's extra features are related to rendering. So you could just use State directly if your module doesn't do any rendering. But if your module does need rendering logic, then Component will work better for you.
+
+Many people have their own favorite way of dealing with rendering logic. Some prefer to use template languages that completely separate it from the business logic, while others like to keep both close together in the same file. Metal.js doesn't force developers to go with only one of those. By default it offers integration points with both closure templates from Google and JSX from Facebook, and it's possible to add more options, since the rendering layer is customizable.
```
Comment (blzaugg) [inline src/pages/docs/getting-started/index.md:29]: Found at: https://github.com/metal/metaljs.com/search?utf8=%E2%9C%93&q=own+favorite+way&type=

File: src/pages/docs/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+###### {$page.description}
+
+<article id="1">
+
+## Introduction
+
+There are many different ways to build components using Metal.js. You can use the built-in integrations with [Soy](http://developers.google.com/closure/templates/) or [JSX templates](https://facebook.github.io/jsx/), or even build your own template abstraction on top of Metal.js and use it instead.
+
+You can build the ES6 code using babel, webpack or any other package bundler. You can test using karma, jasmine or any other test framework, and even use any directory structure for your project. To sum it up, you can customize everything to your own needs. The world is your oyster.
```
Comment (blzaugg) [inline src/pages/docs/getting-started/index.md:17]: Found at: https://github.com/metal/metaljs.com/search?utf8=%E2%9C%93&q=build+the+ES6+code+using+babel&type=

File: src/pages/docs/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+</article>
+
+<article id="3">
+
+## Introducing Soy
+
+Closure Templates are a client- and server-side templating system that helps you dynamically build reusable HTML and UI elements. They have a simple syntax that is natural for programmers, and you can customize them to fit your application's needs. In contrast to traditional templating systems, in which you must create one monolithic template per page, you can think of Closure Templates as small components that you compose to form your user interface. You can also use the built-in message support to easily localize your applications.
+
+Closure Templates are implemented for both JavaScript and Java, so that you can use the same templates on both the server and client side. They use a data model and expression syntax that work for either language. For the client side, Closure Templates are precompiled into efficient JavaScript.
+
```
Comment (blzaugg) [inline src/pages/docs/getting-started/index.md:40]: May be covered at: https://github.com/metal/metaljs.com/blob/35b6296819c9dc3dd28a3579eefd3c4fda81a071/src/pages/docs/guides/soy-components.md

File: src/pages/docs/getting-started/index.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+</article>
+
+<article id="4">
+
+## Introducing JSX
+
+JSX is a XML-like syntax extension to ECMAScript without any defined semantics. It may remind you of a template language, but it comes with the full power of JavaScript.
+
+</article>
```
Comment (blzaugg) [inline src/pages/docs/getting-started/index.md:49]: May be covered at: https://github.com/metal/metaljs.com/blob/35b6296819c9dc3dd28a3579eefd3c4fda81a071/src/pages/docs/guides/jsx-components.md

File: src/pages/docs/getting-started/index.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+<article id="6">
+
+## Trying it out
+
+This guide will focus on a quick and easy way to get started with Metal.js. But if you are excited to try it out as soon as possible you can just play around with this [JSFiddle](https://jsfiddle.net/metaljs/y1tqa7vz/).
+
```
Comment (blzaugg) [inline src/pages/docs/getting-started/index.md:56]: Found at: https://github.com/metal/metaljs.com/search?utf8=%E2%9C%93&q=jsfiddle&type=

File: src/pages/docs/getting-started/yeoman-generator.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+###### {$page.description}
+
+<article id="6">
+
+## Using Yeoman Generator
+
+You can organize your Metal.js project in any way you want, but to start out we recommend using the [Yeoman generator](http://npmjs.com/package/generator-metal) we've created, which prepares both the project structure as well as a basic development workflow for you.
+
```
Comment (blzaugg) [inline src/pages/docs/getting-started/yeoman-generator.md:16]: Found at: https://github.com/metal/metaljs.com/blob/35b6296819c9dc3dd28a3579eefd3c4fda81a071/src/pages/docs/guides/yeoman-generator.md

File: src/pages/docs/getting-started/index.md (hunk 3 above)
Comment (blzaugg) [reply src/pages/docs/getting-started/index.md:40]: These paragraphs are not directly addressed in the migration.
- [ ] Are they valuable info?
- [ ] Should we integrate them when we review the Metal content?

File: src/pages/docs/getting-started/index.md (hunk 4 above)
Comment (blzaugg) [reply src/pages/docs/getting-started/index.md:49]: This paragraph is not directly addressed in the migration.
- [x] Is it valuable info?
- [x] Should we integrate it when we review the Metal content?

**Comment by robframpton** [comment]
@blzaugg can we close this?

File: src/pages/docs/getting-started/index.md (hunk 3 above)
Comment (blzaugg) [reply src/pages/docs/getting-started/index.md:40]: @Robert-Frampton, I'd like these answered first.

File: src/pages/docs/getting-started/index.md (hunk 4 above)
Comment (blzaugg) [reply src/pages/docs/getting-started/index.md:49]: @Robert-Frampton, I'd like these answered first.

Comment (robframpton) [reply src/pages/docs/getting-started/index.md:49]: Hey @blzaugg 

I think this is useful info, can we cherry-pick this paragraph into the appropriate existing page?

**Comment by jbalsas** [comment]
@blzaugg, is this accounted for in the migration?

File: src/pages/docs/getting-started/index.md (hunk 4 above)
Comment (blzaugg) [reply src/pages/docs/getting-started/index.md:49]: We can.

**Comment by blzaugg** [comment]
The noted content (with comment checkboxes) will be migrated at a later time.

I'd like to keep this issue open or open an new PR/issue with the noted content.

**Comment by blzaugg** [comment]
@Robert-Frampton the noted content needs to be followed up on.

**Pull request closed without merging**